motion/README.md

```diff
@@ -0,0 +1,122 @@
+It currently implements three MPC protocols for two or more parties: arithmetic and Boolean GMW as well as the BMR protocol, 
+which is the multi-party version of the constant-round Yao's garbled circuit protocol.
+All three protocols provide security against all but one passively corrupted parties, i.e., parties are supposed to follow the protocol but try to learn additional information from what they observe in the protocol.
+MOTION allows to convert between all three protocols and provides both high-level interfaces such as secure integers as well as low-level APIs 
+for oblivious transfer and fine-grained control over MOTION's internal mechanisms.
+Two design aspects, asynchronous circuit evaluation and serialized communication, enable very flexible use of protocols and primitives, e.g.,
+it is possible to use multiple different flavors of oblivious transfer and various MPC protocols all in parallel, which will wait for particular events to happen, 
+without any need of manual synchronization. 
+
+MOTION was developed by Lennart Braun (Aarhus University), Daniel Demmler (Universität Hamburg), 
+Thomas Schneider and Oleksandr Tkachenko ([ENCRYPTO group](https://encrypto.de), TU Darmstadt).
+
+Our recommendation: MOTION provides a powerful low-level crytographic interface that gives the developer significant control over performance. 
+We recommend MOTION for users who are familiar with MPC protocols, their relative tradeoffs, and the circuit model of computation.
+Furthermore, MOTION is well-suited for (non-invasive) prototyping and mixing of (new) protocols and conversions due to its asynchronous design. 
+More details can be found in [our paper](https://ia.cr/2020/1137).
+
+## Docker setup
+
+Create a Docker image. This will take a few minutes. You only have to do this once.
+```
+$ docker build -t motion .
+```
+Spin up a Docker container from the image.
+```
+$ docker run -it --rm motion 
+```
+Please note that any changes you make in the container are not persistent.
+
+Note: a Dockerfile as well as the provided examples are also contained in the [original MOTION repo](https://encrypto.de/code/MOTION) and very similar instructions can be used 
+to build a Docker container from the original repo directly. See the [MOTION README](https://github.com/encryptogroup/MOTION/blob/master/README.md) for more details. 
+
+## Architecture
+
+MOTION is designed as a C++ library. It defines several classes that
+implement the MPC protocols. It is written in an object-oriented way. 
+Secure data is currently limited to unsigned C integer types.
+Support for floating and fixed-point operations is under active development. 
+MOTION supports SIMD operations for efficient parallelization.
+MOTION is significantly more memory and running time efficient when using SIMD 
+and needs slightly less communication due to better packing of messages, so we strongly encourage to use it.
```

Comment:
```suggestion
MOTION supports SIMD operations for efficient parallelization.
Using SIMD leads to higher efficiency in terms of memory and run time, and also slightly less communication due to better packing of messages.
```

Also, please add instructions how it can be enabled (is there a compiler option or something comparable that enables this feature?)